### ISSUE TITLE
More Complex orientdb query. 

### ISSUE THREAD

**Issue by martingg88** [vjsrinath/sails-orientdb#5]
if i want to do more complicated query like fetchplan, traverse and so on. 

can i user the following method to achieve what i expect?

this.table.query('complicated query statement here');

thanks

**Comment by vjsrinath** [comment]
Yes, you can use it. If you want access to underlying db object used by the adapter you can get it by calling, 

Collection.getDB(function(db){
//you can start calling oriento db object api directly from here
});

**Comment by martingg88** [comment]
thanks...
